### ISSUE TITLE
integration of panels

### ISSUE THREAD

**Issue by stephanos** [suda/tool-bar#90]
Hey there,

I just tried out VisualStudio Code and absolutely loved the idea of having a side bar which toggles between various panels:

<img width="84" alt="screen shot 2015-09-14 at 21 00 55" src="https://cloud.githubusercontent.com/assets/159852/9858962/04f8a8ee-5b24-11e5-91d4-c602618cc020.png">

So the first one shows the file tree, selecting the second one gives you a search panel:

<img width="382" alt="screen shot 2015-09-14 at 21 06 04" src="https://cloud.githubusercontent.com/assets/159852/9859023/70ba148c-5b24-11e5-99f8-8bdaf4b5b68a.png">

The other ones are the GitHub integration and the debugging tools.

Long story short, I was wondering if this would make sense for Atom. I think it would be great to have something like a panel switcher. I especially love that they can show relevant meta info (e.g. how many files changed) on the icons.

I suppose I just wanted to hear your thoughts on this :)

**Comment by jerone** [comment]
Related to https://github.com/suda/tool-bar/issues/43

**Comment by jerone** [comment]
https://github.com/suda/tool-bar/issues/43
